src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -339,5 +339,16 @@ trait MarathonConf
     default = Some(GpuSchedulingBehavior.Undefined),
     validate = validateGpuSchedulingBehavior
   )
+
+  //validations
+
+  //  we must set gc actor threshold to not trigger it too early, 2 * max deployments looks like a good default
```

Comment:
What is the justification behind `2 * max deployments` formula? Why not `max deployments + 5`?

Reply:
because every deployment is a `root group before` + `root group after`.